src/lib.rs

```diff
@@ -847,10 +848,10 @@ impl<'a> UncompressedHeader for BitWriter<'a, BE> {
         }
       } else { // Inter frame info goes here
         if fi.intra_only {
```

Comment:
I think you'd like to do:
assert!(fi.refresh_frame_flags != 0xff);
given av1 spec that: 
"If frame_type is equal to INTRA_ONLY_FRAME, it is a requirement of bitstream conformance that refresh_frame_flags is not equal to 0xff."